test/src/sender/main.cpp

```diff
@@ -1,191 +1,199 @@
-        generator_con, generator_ack, sender1
-    };
-    cadmium::dynamic::modeling::EICs eics_TOP = {};
-    cadmium::dynamic::modeling::EOCs eocs_TOP = {
-        cadmium::dynamic::translate::make_EOC
-            <sender_defs::packet_sent_out,outp_pack>("sender1"),
-            cadmium::dynamic::translate::make_EOC
-                <sender_defs::ack_received_out,outp_ack>("sender1"),
-                cadmium::dynamic::translate::make_EOC
-                    <sender_defs::data_out,outp_data>("sender1")
-    };
-    cadmium::dynamic::modeling::ICs ics_TOP = {
-        cadmium::dynamic::translate::make_IC
-            <iestream_input_defs<Message_t>::out,
-                sender_defs::control_in>("generator_con","sender1"),
-            cadmium::dynamic::translate::make_IC
-                <iestream_input_defs<Message_t>::out,
-                    sender_defs::ack_in>("generator_ack","sender1")
-    };
-    std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP =
-        std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
-                                                                    "TOP",
-                                                                    submodels_TOP,
-                                                                    iports_TOP,
-                                                                    oports_TOP,
-                                                                    eics_TOP,
-                                                                    eocs_TOP,
-                                                                    ics_TOP);
-
-    ///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration
-        <double,std::ratio<1>>> (hclock::now() - start).count();
-    cout<<"Model Created. Elapsed time: "<<elapsed1<<"sec"<<endl;
-    
-    cadmium::dynamic::engine::runner<NDTime, logger_top> r(TOP, {0});
-    elapsed1 = std::chrono::duration_cast<std::chrono::duration
-        <double, std::ratio<1>>> (hclock::now() - start).count();
-    cout<<"Runner Created. Elapsed time: "<<elapsed1<<"sec"<<endl;
-
-    cout<<"Simulation starts"<<endl;
-
-    r.run_until(NDTime("04:00:00:000"));
-    auto elapsed = std::chrono::duration_cast<std::chrono::duration
-        <double, std::ratio<1>>> (hclock::now() - start).count();
-    cout<<"Simulation took:"<<elapsed<<"sec"<<endl;
-    return 0;
-}
+#include <iostream>
+#include <chrono>
+#include <algorithm>
+#include <string>
+
+#include <cadmium/modeling/coupling.hpp>
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/dynamic_model_translator.hpp>
+#include <cadmium/concept/coupled_model_assert.hpp>
+#include <cadmium/modeling/dynamic_coupled.hpp>
+#include <cadmium/modeling/dynamic_atomic.hpp>
+#include <cadmium/engine/pdevs_dynamic_runner.hpp>
+#include <cadmium/logger/tuple_to_ostream.hpp>
+#include <cadmium/logger/common_loggers.hpp>
+
+
+#include "../lib/DESTimes/include/NDTime.hpp"
```

Comment:
I tried running the program for unit tests but it didn't go through. Please check the error in the screenshot. I understand that the error is due to the incorrect relative path of NDTime.hpp in the #include statements. 
It should be "#include "../../../lib/DESTimes/include/NDTime.hpp" . I made the changes in the changes in my local and tried running the code and it was working fine. Please check on the relative path and make the corrections. 
![relative path error](https://user-images.githubusercontent.com/44588557/63225451-3930be00-c19e-11e9-906d-294fe6c8b9b1.PNG)

Reply:
thanks for catching it 
it has been solved